# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/file_access_controls.html
# Commit: 6fd57b83bb52
# Full Hash: 6fd57b83bb52e9683bade8cbfb5f8ba6575ac6af
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-02-07 03:55:51
# Regressor Bug: 1582832
# File Overlap Count: 3
# Description:
#   Bug 1582832: Part 1 - Make FrameLoader owner rather than DocShell responsible for discarding a BC. r=nika
#   
#   There are all sorts of lifecycle issues which arise from making DocShell
#   responsible for discarding BrowsingContexts. In this particular bug, we tend
#   to run into them in cases where we create a BrowsingContext for a FrameLoader,
# ==============================================================================

diff -r 28a9e4b075af -r 6fd57b83bb52 dom/media/test/file_access_controls.html
--- a/dom/media/test/file_access_controls.html	Thu Feb 06 18:35:10 2020 +0000
+++ b/dom/media/test/file_access_controls.html	Thu Feb 06 19:07:56 2020 +0000
@@ -2,7 +2,7 @@
 <head>
   <script type="text/javascript" src="manifest.js"></script>
 </head>
-<body onload="setTimeout(load, 0);" onunload="done()">
+<body onload="setTimeout(load, 0);">
 <script>
 
 // Page URL: http://example.org/tests/dom/media/test/file_access_controls.html
@@ -115,6 +115,7 @@
     } else {
       //dump("Exiting...\n");
       // We're done, exit the test.
+      done();
       window.close();
       return;
     }